eos-core-depends

```diff
@@ -58,3 +58,94 @@ unoconv
 unrar
 whiptail
 xdg-app
+
+
+# moved here from os-depends due to installer
+chromium-browser
+cups
+eos-social
+eos-speedwagon
+eos-tech-support
+eos-tutorial
+eos-updater
+evince
+fake-hwclock
+file-roller
+fonts-arabeyes
+fonts-arphic-ukai
+fonts-arphic-uming
+fonts-cabinsketch
+fonts-crosextra-caladea
+fonts-crosextra-carlito
+fonts-dejavu
+fonts-dosis
+fonts-droid
+fonts-dustin
+fonts-farsiweb
+fonts-gargi
+fonts-indic
+fonts-kacst
+fonts-khmeros
+fonts-knda
+fonts-lato
+fonts-nanum
+fonts-nanum-coding
+fonts-ocr-a
+fonts-paktype
+fonts-quattrocento
+fonts-roboto
+fonts-sil-andika
+fonts-sil-padauk
+fonts-thai-tlwg
+fonts-tlwg-waree
+fonts-unfonts-core
+fonts-uralic
+fonts-vlgothic
```

Comment:
Basically, we just need to make sure that we have a font that covers every supported language, which currently would be the following:
- ar
- bn
- en
- es
- fr
- hi
- id
- pt_BR
- th
- vi
- zh_CN

If we do trim the list of fonts, we will just need to keep in mind that whenever we add a new language we would have to make sure that a font is available.  Perhaps there is a compromise where we keep any fonts that are added for language support, but remove those that are added for apps designs.  I can try to figure out that list.